templates/components/deployments/typescript/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:20-alpine as build
+
+WORKDIR /app
+
+# Install dependencies
+COPY package.json pnpm-lock.yaml* /app/
+RUN npm install 
+
+# Build the application
+COPY . .
+RUN npm run build
+
+# ====================================
+FROM build as release
+
+RUN chmod +x start.sh
+
+CMD ["/bin/sh", "start.sh"]
```

Comment:
i would just support production in the dockerfile:
1. copy only the build output from the `build` layer
2. run `npm start`

Reply:
Supporting dev in Docker, it's a better practice to add a dedicated DEV layer (`FROM build as dev`), but I wouldn't do that for simplicity

Reply:
>Supporting dev in Docker, it's a better practice to add a dedicated DEV layer (FROM build as dev), but I wouldn't do that for simplicity

Yes, i'd better for the front-end image that should include the built static thing only, but it makes the generation more complicated. So let me remove the start script and just run the `npm start` always. Thanks.